### PR TITLE
Bump sarek to 3.0dev

### PR DESCRIPTION
**Pull request by MaxUlysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: containers/snpeff/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 
 # Add conda installation dir to PATH (instead of doing 'conda activate')
-ENV PATH /opt/conda/envs/nf-core-sarek-snpeff-2.7dev/bin:$PATH
+ENV PATH /opt/conda/envs/nf-core-sarek-snpeff-dev/bin:$PATH
```
Comment (FriederikeHanssen) [inline containers/snpeff/Dockerfile:12]: not snpeff-3.0dev?

**Review by FriederikeHanssen — APPROVED** [review]
Looks good. Just the question of whether it should be labeled dev or 3.0dev everywhere. I don't have a preference either way

**Comment by MaxUlysse** [comment]
> Looks good. Just the question of whether it should be labeled dev or 3.0dev everywhere. I don't have a preference either way

Merged before reading comments :facepalm: 

We can always fix that later...

**Comment by FriederikeHanssen** [comment]
Ah no worries. It's a tiny thing about consistency. won't change anything in whether it runs or not.

**Pull request merged**